unstructured_inference/models/detectron2.py

```diff
@@ -21,6 +24,43 @@
 DEFAULT_EXTRA_CONFIG: Final[List[Any]] = ["MODEL.ROI_HEADS.SCORE_THRESH_TEST", 0.8]
 
 
+# NOTE(alan): Entries are implemented as LazyDicts so that models aren't downloaded until they are
```

Comment:
Is there a supported way to download the models ahead of time if you want to do it ahead of time? For instance if you want to include the model in a docker image or something like that?

Reply:
Nothing out of the box. We could do it by running some code that instantiates whichever model we want, or we could build something to do it.

Reply:
As long as you can do something along the lines of what we do for NLTK, we're probably fine 

https://github.com/Unstructured-IO/unstructured/blob/main/Makefile#L37-L38